docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -89,13 +91,207 @@ For the complete list of actions that {ilm} can perform, see <<ilm-actions>>.
 
 [discrete]
 [[ilm-gs-apply-policy]]
-=== Create an index template to apply the lifecycle policy
+=== Create a composable template to create the data stream and apply the lifecycle policy
+
+To create a data stream we'll create a composable template with a `data_stream` definition
+that will specify the lifecycle policy.
+
+For example, you might create a `timeseries_template` that is will bootstrap a data stream
+named `timeseries`.
+
+To enable the ILM to manage the data stream, the template configures one {ilm-init} setting:
+
+* `index.lifecycle.name` specifies the name of the lifecycle policy to apply to the data stream.
+
+You can use the {kib} Create template wizard to add the template.
+This wizard invokes the put _index_template API to create the <<indices-templates,composable index template>>
+with the options you specify.
+
+The underlying request looks like this:
+
+[source,console]
+-----------------------
+PUT _index_template/timeseries_template
+{
+  "index_patterns": ["timeseries"],                 <1>
+  "data_stream": {
+    "timestamp_field": "@timestamp"                  <2>
+  },
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 1,
+      "index.lifecycle.name": "timeseries_policy"     <3>
+    }
+  }
+}
+-----------------------
+// TEST[continued]
+
+<1> Apply the template when a document is indexed into the `timeseries` target.
+<2> Identifies the timestamp field for the data source, a field that must be present
+in all documents indexed into the `timeseries` data stream.
+<3> The name of the ILM policy used to manage the data stream.
+
+You can also invoke this API directly to add templates.
+
+[discrete]
+[[ilm-gs-bootstrap]]
+=== Bootstrap the data stream
+
+To get things started, you simply index a document into the name or wildcard pattern defined
+in the `index_patterns` of the <<indices-templates,composable index template>> that will
+create the data stream.
+{es} will automatically create the corresponding data stream and will index the document into
+the first backing index.
+
+For example, the following request creates the `timeseries` data stream and the first generation
+backing index called `.ds-timeseries-000001`.
+
+[source,console]
+-----------------------
+POST timeseries/_doc
+{
+  "message": "logged the request",
+  "@timestamp": "1591890611"
+}
+
+-----------------------
+// TEST[continued]
+
+When the rollover conditions are met, the `rollover` action:
+
+* Creates the second generation backing index, name `.ds-timeseries-000002`.
+As it is the backing index of the `timeseries` data stream which was created using the `timeseries_template`
+the template configuration will be applied to the new index.
```

Comment:
Nit: Reword for clarity
```suggestion
Because it is a backing index of the `timeseries` data stream, the configuration from the `timeseries_template` composable template is applied to the new index.
```